### PR TITLE
fix(spotify-web): various unthemed elements

### LINKED ISSUE #826: Spotify unthemed background behind sidebar elements
### Is there an existing issue outlining your problem?

- [X] I am using the latest version of the provided userstyle.
- [X] I have searched the existing issues and they do not solve my problem.

### What userstyles are you seeing the problem on?

lbl:spotify-web

### Describe your problem.

The background behind the Home/Search panel and the Your Library panel isn't themed, which is visible at the rounded corners and the gap between panels.

### Attach screenshots.

image

### What browser(s) are you seeing the problem on?

Firefox v125.0.1

### Any additional comments?

_No response_

### LINKED ISSUE #834: spoitfy: unthemed elements
### Is there an existing issue outlining your problem?

- [X] I am using the latest version of the provided userstyle.
- [X] I have searched the existing issues and they do not solve my problem.

### What userstyles are you seeing the problem on?

lbl:spotify-web

### Describe your problem.

cookie banner/modal, cookie settings icon (found on the bottom of the left sidebar), and signup promo banner 

### Attach screenshots.

image

image

image

image

### What browser(s) are you seeing the problem on?

firefox v125.0.1

### Any additional comments?

_No response_

### PR DESCRIPTION
**Pull request by Papweer**
## 🔧 What does this fix? 🔧

Themes the previously unthemed Accept Cookies banner, vendors info popup and the signup banner. Also fixes the left sidebar background on the main page being unthemed.
fixes #834 and also fixes #826

## 🗒 Checklist 🗒

- [x] I have read and followed Catppuccin's [contributing guidelines](https://github.com/catppuccin/userstyles/blob/main/docs/CONTRIBUTING.md).
- [x] I have updated the version appropriately in the `==UserStyle==` header of the `catppuccin.user.css` file.

### CODE REVIEW

**Comment by uncenter** [comment]
Please follow the https://www.conventionalcommits.org/en/v1.0.0/ format for titling your pull request. In this case, the type of change is a `fix`, the scope should be `spotify-web`, and the body should be a very succinct summary of your changes.

File: styles/spotify-web/catppuccin.user.css [hunk 1]
```diff
@@ -295,6 +298,82 @@
...
+      }
+      .ot-pc-footer {
+          border-top-color: @accent-color !important;
+      }
+      #onetrust-pc-sdk li>button {
+          border-top-color: @accent-color !important;
+      }
+      #onetrust-consent-sdk #onetrust-pc-sdk #ot-sel-blk {
+          background-color: @base !important;
+      }
```
Comment (uncenter) [inline styles/spotify-web/catppuccin.user.css:372]: cc @isabelroses I actually themed this for Stack Overflow (https://github.com/catppuccin/userstyles/blob/c977d9e354082ca79e78bc7bec04da0ac2611e41/styles/stack-overflow/catppuccin.user.css#L313-L406), seeing as it looks like a reusable component from across websites maybe we add a global userstyle that includes tweaks like this? Or possibly generate it statically and add `@import` lines for userstyles that involve websites with the OneTrust cookie consent popup?

Comment (isabelroses) [reply styles/spotify-web/catppuccin.user.css:372]: Ohhh this is interesting. Heres my idea. We create a "common" dir and place commonalities between themes and publish them there. But I'll admit this maybe a bit overkill and just repeating the same style here might be easier.

**Comment by Tnixc** [comment]
image
We still have some unthemed elements, we might want to theme those as well before we merge

**Comment by Papweer** [comment]
I themed the unthemed elements mentioned above and a few more. Not sure what to do with the cookie banner theming but for now I just left it in.

**Comment by Tnixc** [comment]
image
image

Glad you fixed the elements above. There's still some minor things to fix here. Make sure you update the version from 0.0.1 to 0.0.2, It seems like its gone back to 0.01 in your latest commit.

**Comment by Papweer** [comment]
I themed the remaining elements. I couldn't find any other unthemed elements but I might have missed some.

**Comment by Papweer** [comment]
I fixed the linting errors. I think it's ready to merge.

**Comment by uncenter** [comment]
@Tnixc would you mind reviewing again? :p

**Comment by Tnixc** [comment]
image

image


Could you fix these modals as well? Sorry for not being through last time.

**Comment by Tnixc** [comment]
image
This too.

**Comment by Papweer** [comment]
Fixed the unthemed elements you mentioned plus some more. I couldn't find anything else but I probably missed something.

**Review by Tnixc — APPROVED** [review]
LGTM 👍. I couldn't find any missing elements.

**Pull request merged**